tcms/templates/navbar.html

```diff
@@ -114,3 +114,5 @@
             </ul>
         </div>
     </nav>
+
+    <span id="defaultPageSize" data-defaultPageSize="{{ SETTINGS.DEFAULT_PAGE_SIZE }}"></span>
```

Comment:
This is a clever idea but you don't need a separate element to hold this information. It is a visual element none the less. Add the data attribute to the `nav` tag and adjust your selectors.